public/app/plugins/panel/geomap/migrations.test.ts

```diff
@@ -55,7 +55,49 @@ describe('Worldmap Migrations', () => {
             "mouseWheelZoom": true,
             "showZoom": true,
           },
-          "layers": [],
+          "layers": [
+            {
+              "config": {
+                "showLegend": true,
+                "style": {
+                  "color": {
+                    "fixed": "dark-green",
+                  },
```

Comment:
It this a mismatch with line 140: `colors: ['#37872D', '#E0B400', '#C4162A', '#8F3BB8'],`

Reply:
Nevermind, I see that in fieldConfig.